packages/api/README.md

```diff
@@ -4,7 +4,33 @@ The HTTP interface implemented as a Cloudflare Worker
 
 ## Getting started
 
-One time set up of your cloudflare worker subdomain for dev:
+We use miniflare to run the api locally, and docker to run ipfs-cluster and postgres with postREST.
+
+```sh
+# Copy <.env.local.tpl> to `.env.local` and fill out the missing worker vars.
+cp .env.local.tpl .env.local
+
+# Install the deps
+npm install
+```
+
+With docker running locally, start all the things:
+
+```
+npm start
+```
+
+🎉 miniflare is running in watch mode; you can save changes to the api code and the worker will update.
+
+Kill the process to stop miniflare, then run `npm run stop` to shutdown the cluster and postgres
+
+```sh
+npm run stop
+```
+
+## Setting up a cloudflare worker
```

Comment:
I'm wondering if this section is still relevant?  Do we want to keep it here in case a developer needs/wants to test the code against a real worker?
If the answer is yes I think we should update this because it might be a bit confusing to a developer without the right context. Also, some of the things in there are not accurate anymore. ie We'd need to update [this](https://github.com/web3-storage/web3.storage/pull/973/files#diff-f3a9540595a860f97e334d7faaa974596dd800c0bb472c65427fddbde50d51b4L77) to launch wrangler dev (not in local mode) instead of miniflare.

If not we could probably get rid of most of this and delete related npm scripts (localtunnel ones)

Reply:
Ok sure.
I'll try to find some spare time to open a PR with some updates to the Readme as follow-up!